hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java

```diff
@@ -1049,6 +1049,21 @@ public MiniHBaseCluster startMiniCluster(int numSlaves) throws Exception {
     return startMiniCluster(option);
   }
 
+  /**
+   * Start up a minicluster of hbase, dfs and zookeeper clusters with given slave node number and
+   * a set of master ports. All other options will use default values, defined in
```

Comment:
"master ports" are not an argument here... I'm missing something?

Reply:
Re-did the patch, not valid anymore.